packages/calendar/Calendar.test.tsx

```diff
@@ -1,17 +1,223 @@
-import { Calendar } from './Calendar';
+import * as expect from 'expect';
+import * as isSameDay from 'date-fns/is_same_day';
+import * as parse from 'date-fns/parse';
+import { Calendar } from './';
```

Comment:
we have to import explicitly from './index' , because some issues with build....